### PR TITLE
Start round of game and calculation results of the round

### PR DESCRIPTION
**Pull request by Evgenia-cyber**

### CODE REVIEW

**Review by Vlad-coder-678 — APPROVED** [review]

File: poker/src/utils/isAllUsersVoted.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { IssueChatItem } from '../types/common';
+
+const isAllUsersVoted = (array: Array<IssueChatItem>): boolean => {
+  const index = array.findIndex((el) => el.value === '-');
+  return index === -1;
```
Comment (AnAtoliy-AA) [inline poker/src/utils/isAllUsersVoted.ts]: return array.every((el) => el.value !== '-');

File: poker/src/utils/updateStatistics.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+const createCardValueAndCountOfVotesArray = (array: Array<IssueChatItem>): Array<TempType> => array
+.reduce((res: Array<TempType>, item) => {
+  if (item.value !== '-') {
+    const index = res.findIndex((el) => el.value === item.value);
+    if (index !== -1) res[index] = { value: item.value, count: res[index].count + 1 };
+    else res.push({ value: item.value, count: 1 });
+  }
+
+  return res;
+}, []);
```
Comment (AnAtoliy-AA) [inline poker/src/utils/updateStatistics.ts:18]: hard to read and understand

File: poker/src/utils/isAllUsersVoted.ts (hunk 1 above)
Comment (Evgenia-cyber) [reply poker/src/utils/isAllUsersVoted.ts]: Fixed

File: poker/src/utils/updateStatistics.ts (hunk 2 above)
Comment (Evgenia-cyber) [reply poker/src/utils/updateStatistics.ts:18]: If there is enough time before the deadline, I will refactor the implementation of updateStatistics function.

**Pull request merged**